lib/logstash/inputs/redis.rb

```diff
@@ -49,6 +49,36 @@ module LogStash module Inputs class Redis < LogStash::Inputs::Threadable
   # TODO: change required to true
   config :key, :validate => :string, :required => false
 
+  # The value of the renamed redis BLPOP command. Note that this uses a
```

Comment:
I suggest adding a `renamed_commands` option that takes a hash instead of explicit renaming of a strict set of commands. This would mean that if we want to add the ability to support renaming of an operation not listed here we'd have to add an extra setting. With a global hash option we could do instead:

```
input {
  redis {
    renamed_commands => {
      "blpop" => "alias1"
      "evalsha" => "alias2"
    }
  }
}
```

Reply:
Also, could you add the documentation of this option also to the `docs/index.asciidoc` as it's the source of truth for the plugin docs